charts/kube-state-metrics/README.md

```diff
@@ -0,0 +1,66 @@
+# kube-state-metrics Helm Chart
+
+Installs the [kube-state-metrics agent](https://github.com/kubernetes/kube-state-metrics).
+
+## Get Repo Info
+
+```console
+helm repo add kube-state-metrics https://kubernetes.github.io/kube-state-metrics
+helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+```console
+# Helm 3
+$ helm install [RELEASE_NAME] kube-state-metrics/kube-state-metrics [flags]
+
+# Helm 2
```

Comment:
Maybe remove the helm2 instructions?

Reply:
Once this has been in, I'll work on removing it (see: https://github.com/kubernetes/kube-state-metrics/pull/1358 )

Reply:
What is the plan, removing the chart from kube-state-metrics.git once this is merged?